### ISSUE TITLE
Invalid type for encryption column

### ISSUE THREAD

**Issue by warrensbox** [mmmorris1975/aws-secrets-sync#2]
We finally figured out what what causing confd to "panic". 
`panic: runtime error: invalid memory address or nil pointer dereference`

Confd is looking for a boolean type here:
`if val.S != nil && ok && *encrypted.BOOL {`
https://gilligan.pearsondev.com/containers/confd/blob/master/backends/dynamodb/client.go#L103

However, in aws-secret-sync, the code sets it to a string - `S` here:
```go
i := dynamodb.PutItemInput{
		TableName: aws.String(b.table),
		Item: map[string]*dynamodb.AttributeValue{
			b.pk:        {S: aws.String(key)},
			"value":     {S: aws.String(data)},
			"encrypted": {S: aws.String("true")},
		},
	}
```
 https://github.com/mmmorris1975/aws-secrets-sync/blob/master/backend_dynamodb.go#L79

I have created a merge request for this fix. 
https://github.com/mmmorris1975/aws-secrets-sync/pull/1

**Comment by warrensbox** [comment]
I produced a new docker image with my change and tested it.
quay.io/warren/test-aws-secret-sync:0.3.2

The test passed and confd is able to decrypt the encrypted values.

**Comment by mmmorris1975** [comment]
Fixed in release 0.3.2
